lib/commons/text/subtree-text.js

```diff
@@ -13,14 +13,32 @@ import getOwnedVirtual from '../aria/get-owned-virtual';
 function subtreeText(virtualNode, context = {}) {
 	const { alreadyProcessed } = accessibleTextVirtual;
 	context.startNode = context.startNode || virtualNode;
-	const { strict } = context;
+	const { strict, inControlContext, inLabelledByContext } = context;
 	if (
 		alreadyProcessed(virtualNode, context) ||
-		!namedFromContents(virtualNode, { strict })
+		virtualNode.props.nodeType !== 1
```

Comment:
Even when subtreeDescendant is true, we need to return an empty string for things that aren't nodes.